ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionBroadcaster.java

```diff
@@ -77,7 +102,7 @@ public void onTransactionsAdded(final Collection<Transaction> transactions) {
       return;
     }
 
-    final int numPeersToSendFullTransactions = (int) Math.ceil(Math.sqrt(currPeerCount));
+    final int numPeersToSendFullTransactions = (int) Math.round(Math.sqrt(currPeerCount));
```

Comment:
This allows for rounding down as well as up, instead of ceil which is up only. It still works, but I'm unclear why it is necessary or preferred?

Reply:
it is not necessary, just seems more appropriate to round instead of ceil, to avoid strange results when few peers are connected, for example with 2 peers, with round we have 1, otherwise is 2.